### PR TITLE
s390x: add LXC support to send DirMaint commands

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by aplanas** [comment]
[Merge the LXC PR](https://trello.com/c/fUi187uY/123-merge-the-lxc-pr)

File: scripts/lib/mkcloud-dirmaint.sh [hunk 1]
```diff
@@ -147,17 +241,55 @@ function dirmaint_do_onhost_deploy_image()
...
+{
+	# assuming that there has been added a CLDPROT PROTODIR entry to
+	# dirmaint that sets common defaults.
+	# The prototype contains default disk and network. If different defaults
+	# are desired, we can also add several prototypes or finalize a user
+	# later on.
+        local node=$1
+        local cldprot=${cldprot:-CLDPROT}
+	# FIXME: I guess, for production, we would use AUTOONLY as password.
+	$lxc -c "cms dirm add $node like $cldprot PW lin390"
```
Comment (aplanas) [inline scripts/lib/mkcloud-dirmaint.sh]: do we want this PW here? Is this not part of .lxccfg?

File: scripts/lib/mkcloud-dirmaint.sh [hunk 2]
```diff
@@ -3,6 +3,59 @@
...
 
+shutdowntime=60
+zvmname=$(vmcp q userid | sed 's/ .*//')
+
+function dirmaint_do_sanity_checks()
+{
+    if [ ! -e ~/.lxccfg ] ; then
+        complain 1 "This script needs lxc configured and ~/.lxccfg to work correctly"
+    else
+	. ~/.lxccfg
```
Comment (bmwiedemann) [inline scripts/lib/mkcloud-dirmaint.sh]: should use spaces for indention instead of tabs

Comment (aplanas) [inline scripts/lib/mkcloud-dirmaint.sh]: Done, thanks!

**Review by dirkmueller — DISMISSED** [review]

**Comment by dirkmueller** [comment]
retriggered gating

**Comment by jdsn** [comment]
@aplanas Is this PR still valid? Needs rebasing and some fixes to make the checks happy.

**Comment by aplanas** [comment]
@jdsn thanks, I will rebase

**Comment by dirkmueller** [comment]
@aplanas can you please rebase?

**Pull request closed without merging**